tests/project_directory_test.php

```diff
@@ -107,4 +107,20 @@ public function testGetDirectoryAvoidPluginsDirectory()
         $this->assertInstanceOf('\\Dxw\\Whippet\\ProjectDirectory', $result->unwrap());
         $this->assertEquals($dir, $result->unwrap()->__toString());
     }
+
+    public function testGetDirectoryWithEnv()
+    {
+        $dir = $this->getDir();
+
+        putenv('WHIPPET_PROJECT_DIRECTORY=/etc');
```

Comment:
Out of curiosity: this looks like it should leave files hanging around in my /etc directory, but it doesn't - how does that work?

Normally for this sort of thing I'd isolate it by creating a `tests/temp` directory and deleting that at the end, but whatever works.